screens/WalletDetailScreen.tsx

```diff
@@ -197,6 +235,34 @@ const WalletDetailScreen = ({
       ? null
       : amountDecimal;
 
+  type tokenProps = { tokenId: string };
+
+  const HideButton = ({ tokenId }: tokenProps) => (
+    <VisibilityArea>
+      <VisibilityText
+        onPress={() => {
+          addTokenToBlacklist(tokenId);
+        }}
+      >
+        Visible in Vault
+      </VisibilityText>
+      <EyeIconImage source={EyeOpenIcon} />
+    </VisibilityArea>
+  );
+
+  const ShowButton = ({ tokenId }: tokenProps) => (
+    <VisibilityArea>
+      <VisibilityText
+        onPress={() => {
```

Comment:
Right now the icon itself is unresponsive to press events.  

The common ways to handle this are outlined here
https://facebook.github.io/react-native/docs/handling-touches.html#touchables
This app is using `TouchableOpacity` in all cases right now, so I would recommend continuing that route for visual consistency.
As an added bonus TouchableOpacity will also bring visual feedback to the touch. 

Example used within app
https://github.com/Bitcoin-com/badger-mobile/blob/develop/components/CoinRow/CoinRow.tsx#L20

Reply:
resolved by using icon only.

Reply:
Touchable opacity is for the visual feedback, added this change in a PR.